x-pack/test/encrypted_saved_objects_api_integration/fixtures/api_consumer_plugin/server/index.ts

```diff
@@ -80,11 +80,9 @@ export const plugin: PluginInitializer<void, void, PluginsSetup, PluginsStart> =
 
         try {
```

Comment:
I know alerting is about to become a consumer with its own set of tests, but can we add a test to this suite ensuring that ESOs work with hidden types as expected?

Reply:
Sure 👍